spec/expression.dd

```diff
@@ -2179,6 +2179,12 @@ $(GNAME AssertArguments):
     )
     )
 
+    $(RATIONALE
+    The rationale for and proper use of `assert` comes up now and then.
+    Here is a long discussion of it:
```

Comment:
At a minimum, "comes" should be "come" (rationale AND use). Pedantically, "usage" is more appropriate than "use" here (*how* something is used vs. the fact that it is used). 

I'd suggest a rephrasing it all as:

"The rationale for and proper usage of assert are sometimes misunderstood. A relevant discussion can be found in the following forum thread:"

Reply:
done